### PR TITLE
v92.3.0

### PR DESCRIPTION
**Pull request by yarneo**
# 92.3.0

This minor release introduces new color properties to TextFields and removes max size constraints on ActivityIndicator.

## New features

MDCTextInputControllerBase now allows setting the input field's border in the resting state using `borderStrokeColor`:

```objc
MDCTextField *textField = [[MDCTextField alloc] init];
MDCTextInputControllerOutlined *controller =
  [[MDCTextInputControllerOutlined alloc] initWithTextInput:textField];
controller.borderStrokeColor = UIColor.redColor;
```

MDCTextInputControllerFloatingPlaceholder now allows setting the error color of the border of the input field while being in edit mode using `floatingPlaceholderErrorActiveColor`:

```swift
let textFieldFilled = MDCTextField()
let textFieldControllerFilled = MDCTextInputControllerFilled(textInput: textFieldFilled)
textFieldControllerFilled.floatingPlaceholderErrorActiveColor = .red
```

### ActivityIndicator

* [Removes 72dp radius limit on activity indicator (#8534)](https://github.com/material-components/material-components-ios/commit/4d39b55d09bdbc13d9338a61f8cb5daeae2b4e83) (Josh Converse)

### Tabs

* [Add TabBarView API to set content padding (#8529)](https://github.com/material-components/material-components-ios/commit/993eea2e9ea3dea8015dba03a8d043d7406ba7f0) (Robert Moore)

### TextFields

* [Add assistive label view (#8539)](https://github.com/material-components/material-components-ios/commit/f06651ee89e8f5b04b3d3a3431713649d12f3766) (Andrew Overton)
* [Add some MDCTextControl properties and some small methods I forgot to implement (#8535)](https://github.com/material-components/material-components-ios/commit/4eb5b58a9625081c2606e94ef10b4c1d25970283) (Andrew Overton)
* [Adding border stroke color (#8510)](https://github.com/material-components/material-components-ios/commit/b52b0b41188bcc8c185b228a68a96fe9cbdbf397) (Galia Kaufman)
* [Adding floatingPlaceholder Error Active color API to controller (#8526)](https://github.com/material-components/material-components-ios/commit/a1c2e4b0a9ad8c76d976f94940a83aa9c7a7e804) (Galia Kaufman)
* [Bring more of base positioning reference over (#8530)](https://github.com/material-components/material-components-ios/commit/9d814f27fc334165365f67313fe18b8a82abecee) (Andrew Overton)
* [Minor cleanup (#8543)](https://github.com/material-components/material-components-ios/commit/de9f2baadfe8184d314a1050035f32b616b96881) (Andrew Overton)

## Multi-component changes

* [Added section for iOS 13 collection considerations (#8527)](https://github.com/material-components/material-components-ios/commit/302077c4ba3a9c95682de3b9e9fd90e0ee1dcb9d) (Randall Li)

---

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,25 +1,3 @@
...
-# block the release and work with the releaser to ensure that the change needs
-# to be propagated from the `develop` branch to the `stable` branch. In nearly
-# all cases, it should not be propagated from `develop` to `stable`.
-#
-# If you are a releaser and see this file change and you're not sure why, you
-# might have accidentally skipped [setting the correct
-# driver in your cloned
-# repository](https://github.com/material-components/material-components-ios/blob/develop/contributing/releasing.md#configure-the-merge-strategy-for-gitattributes).
-# If that's the case, please either revert the accidental change manually or
-# restart the release with a fresh clone and the correct driver.
```
Comment (jverkoey) [inline .gitattributes]: @romoore you may be the only person who understands what's going on with this file on the team. Can you help us understand whether this change is ok?

**Review by jverkoey — APPROVED** [review]
LGTM, pending @romoore's input on the .gitattributes file.

File: .gitattributes (hunk 1 above)
Comment (romoore) [reply .gitattributes]: It should never change. Never ever.


>   DO NOT CHANGE THIS FILE

Comment (romoore) [reply .gitattributes]: The other, I hope, helpful comment:

>   Do not merge this version into `stable`.

Comment (romoore) [reply .gitattributes]: If you have suggestions on better comments, I'd be happy to approve a PR for them.

Comment (jverkoey) [reply .gitattributes]: Thank you Rob! @yarneo to revert.

Comment (yarneo) [reply .gitattributes]: Thanks Rob, reverted file.

**Comment by romoore** [comment]
Looks like this broke Git LFS in `develop`.
https://github.com/material-components/material-components-ios/blob/742f7cf/.gitattributes

**Comment by yarneo** [comment]
Updated. We will need to change the release process and our tooling for this.

**Comment by romoore** [comment]
Yep, that's the goal of https://github.com/material-components/material-components-ios/issues/8392

**Pull request merged**